File: condicoes/1043.py
#1043 TRIÂNGULO
valores = input().split(" ")
valorA = float(valores[0])
valorB = float(valores[1])
valorC = float(valores[2])

if (valorA + valorB) > valorC and (valorB + valorC) > valorA and (valorA + valorC) > valorB:
    perimetro = valorA+valorB+valorC
    print("Perimetro = {:.1f}" .format(perimetro))
else:
    area = ((valorB + valorA)/2)*valorC
    print("Area = {:.1f}" .format(area))

File: javascript/1043.js
/*Triângulo*/
let input = require("fs").readFileSync("/dev/stdin", "utf8");

let lines = input.split(" ");

let A = Number(lines.shift());
let B = Number(lines.shift());
let C = Number(lines.shift());

if(A > B && A > C){
  Maior = A;
}else if(B > C){
  Maior = B;
}else{
  Maior = C;
}
if(Maior == A){
  Soma = B + C;
}else if(Maior == B){
  Soma = A + C;
}else{
  Soma = A + B;
}
if(Soma > Maior){
  Valor = (A + B + C);
  console.log("Perimetro = " + Valor.toFixed(1));
}else{
  Resultado = ((A + B) * C) /2;
  console.log("Area = " + Resultado.toFixed(1));
}